lib/mayaUsd/render/mayaToHydra/utils.cpp

```diff
@@ -56,6 +56,18 @@ global proc {{override}}OptionBox() {
     attrControlGrp -label "Show Wireframe on Selected Objects" -attribute "defaultRenderGlobals.mtohWireframeSelectionHighlight" -changeCommand $cc;
     attrControlGrp -label "Highlight Selected Objects" -attribute "defaultRenderGlobals.mtohColorSelectionHighlight" -changeCommand $cc;
     attrControlGrp -label "Highlight Color for Selected Objects" -attribute "defaultRenderGlobals.mtohColorSelectionHighlightColor" -changeCommand $cc;
+)mel"
+#if USD_VERSION_NUM >= 2005
+R"mel(
+    attrControlGrp -label "Highlight outline (in pixels, 0 to disable)" -attribute "defaultRenderGlobals.mtohSelectionOutline" -changeCommand $cc;
+)mel"
+#endif
+#if USD_VERSION_NUM >= 1911
+R"mel(
+    attrControlGrp -label "Enable color quantization" -attribute "defaultRenderGlobals.mtohColorQuantization" -changeCommand $cc;
```

Comment:
Maybe change this to "Enable color quantization for selection-coloring", since that's a bit more descriptive.

Alternatively... I'd be fine if we didn't even make this an option, and just left it always off.

Reply:
The color quantization is on the frame buffer as a whole, it just happens to run in the 'colorizeTask'.
It's in 19.11, 20.02, and 20.05, so not sure how soon it's going to be deprecated.

Reply:
Oh, really? Yuck... maybe I'm not understanding what this is doing, or why it's there, then - my knowledge of color science / color management is limited.  But I'm not sure in what situations you'd want this on...?

Reply:
It does two things, a quantization/dithering (so that banding is less apparent) and conversion from linear->sRGB.

I imagine tying into Maya's color-management will eventually be the way to handle such things, but until then I'd like an easy way to make sure this is off.